dev-lua/lua-openssl/files/lua-openssl-0.4.1-Makefile.patch

```diff
@@ -0,0 +1,51 @@
+diff --git a/Makefile b/Makefile
```

Comment:
Okay, I’ve updated it. However, to be honest, I’m not sure if this patch is really needed/useful. I just saw this in other Lua ebuilds. I _think_ that the rationale behind it is to respect flags defined in the system environment. (I have little experience with C stuff)